addons/website_gengo/static/src/xml/website.gengo.xml

```diff
@@ -6,6 +6,7 @@
     <a role="button" class="btn btn-danger d-none gengo_inprogress disabled"  href="#"> <i class="fa fa-clock-o"></i> Translation in Progress</a>
     <a role="button" class="btn btn-link gengo_info" data-action="translation_gengo_info">Count Words</a>
 </t>
+<!-- TODO: This is not working since https://github.com/odoo/odoo/commit/2972976962617d4b8a0113bae58c640ab41cdff8 ? Try to override web_editor -->
```

Comment:
Need to be discussed

Reply:
Need to discuss?

Reply:
As discussed, this website_gengo template overriding issue is now written in your wishlist pad of todo things :pray: 
I will wisely remove that comment from my PR

Reply:
Actually I fixed it in a separate commit in this PR.
Maybe it should land in 11.0 and not in this PR in 12.0?

Reply:
If stable, I suppose yes.

Reply:
Alright, I extracted the commit in 11.0 at https://github.com/odoo/odoo/pull/29955
This one is LGTM
Thanks for the review